deps.sh

```diff
@@ -42,23 +42,25 @@ YUMCMD="yum"
 if [ $distro = "Kali" ]
 then
     APT_CANDIDATES="git build-essential libqt4-opengl mtd-utils gzip bzip2 tar arj lhasa p7zip p7zip-full cabextract util-linux firmware-mod-kit cramfsswap squashfs-tools zlib1g-dev liblzma-dev liblzo2-dev sleuthkit default-jdk lzop cpio"
```

Comment:
Kali should also be updated: libqt4-opengl is no more available and should also be replaced by qt5base-dev

Reply:
Applied change.